tools/kdc/setup_kdc_db.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/expect
+
+spawn /usr/local/sbin/kdb5_util create -s -r $env(REALM)
```

Comment:
According to https://linux.die.net/man/8/kdb5_util one could use the following option:
```
-P password
specifies the master database password. This option is not recommended.
```